src/Payum/Stripe/StripeCheckoutGatewayFactory.php

```diff
@@ -16,7 +16,7 @@ class StripeCheckoutGatewayFactory extends GatewayFactory
      */
     protected function populateConfig(ArrayObject $config)
     {
-        if (!class_exists('Stripe')) {
+        if (!class_exists('\Stripe\Stripe')) {
             throw new \LogicException('You must install "stripe/stripe-php" library.');
```

Comment:
Don't see why not :)
Also, I'm pretty sure that there are not breaking changes from 2.0 - 3.6, for this limited usage. So there is no need to make this so tightly coupled to 3.6. Updating to support that range. (Namespaces were introduced in 2.0)